pkg/comp-functions/functions/vshnnextcloud/deploy.go

```diff
@@ -428,3 +443,12 @@ func setBackgroundJobMaintenance(t vshnv1.TimeOfDay, nextcloudConfig string) str
 	backgroundJobHour := t.GetTime().Add(40 * time.Minute).Add(time.Hour).Hour()
 	return strings.Replace(nextcloudConfig, "%maintenance_value%", strconv.Itoa(backgroundJobHour), 1)
 }
+
+func validateFQDNs(fqdns []string) error {
+	for _, fqdn := range fqdns {
+		if !valid.IsDNSName(fqdn) {
```

Comment:
Idea: this is something we could check in a webhook, so it doesn't break silently for the user.

Reply:
I've refactored the code and moved this validation to the webhook